server/src/main/java/com/octopus/teamcity/opentelemetry/server/TeamCityBuildListener.java

```diff
@@ -28,19 +30,22 @@
 
     private final OTELHelper otelHelper;
     private static final String ENDPOINT = TeamCityProperties.getProperty(PluginConstants.PROPERTY_KEY_ENDPOINT);
+    private final ConcurrentHashMap<String, Long> checkoutTimeMap;
 
     @Autowired
     @SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection")
     public TeamCityBuildListener(EventDispatcher<BuildServerListener> buildServerListenerEventDispatcher) {
```

Comment:
Is it a thing in Java that one constructor can call another?
That way, we can avoid the bug magnet of having to duplicate the code in each

Reply:
ah great idea! I'll look into that one actually.